tools/pl-paths.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Extract all the Pattern Lab paths out of built PL assets
+ */
+
+const {
+  patternPaths,
+} = require('../dist/pl/styleguide/data/patternlab-data.json');
+
+// Shape of data:
+//
+// patternPaths: {
+//   protons: {
+//     borders: '00-protons-demo-borders',
+//     tables: '00-protons-demo-tables',
+//     // ...
+//   },
+//   atoms: {},
+//   molecules: {},
+//   // ...
+// }
+
+const paths = [].concat(
+  ...Object.values(patternPaths).map(atomic =>
+    Object.values(atomic).map(path => path)
```

Comment:
I think you can avoid the second map call here: `Object.values(atomic).map(path => path)` if you just access the value directly using `atomic` as the key: `Object.values(patternPaths[atomic])`.

Reply:
Oh nvm, you are grabbing both `Object.values`.

Reply:
I don't think so since the second Object.values() is returning an array of all atomic values, ie `['00-protons-demo-borders', '00-protons-demo-tables', ...]`;

Reply:
I want to revisit this.